### DISCUSSION TITLE
How to enable ad hoc server name

### DISCUSSION THREAD

**Discussion by majkinetor** [1Remote/1Remote#453] [Q&A] [answered]
I can't remember, I know we had this at some point.

I have a fleet of servers named server1... server20 as GitLab runners. When some of those fail I need to RDP. 

@VShawn, How can I add one connection and just change hostname after starting the connection via popup? :)

**Comment by majkinetor** [comment, marked as answer]
OK, I used this, its quick

![image](https://github.com/1Remote/1Remote/assets/85767/d2270e0b-b446-4a10-ae05-a06f5233206c)

Not really good option if you have 100 servers though or random names ...

**Reply by majkinetor** [reply]
Ah god damnit I forgot no 6, and there is no way to insert it in the middle, now I have to move all :)

**Reply by majkinetor** [reply]
Maybe display actuall alternative name in the title ?

![image](https://github.com/1Remote/1Remote/assets/85767/d03946b0-6adf-4f0d-a81e-e51cdf1045d1)

**Comment by VShawn** [comment]
![image](https://github.com/1Remote/1Remote/assets/10143738/e14ed15b-e839-4e37-b0e4-15a2ebd9b667)

You can find it here on the launcher.

**Reply by VShawn** [reply]
This is my case, `31` is my server's name, the alternative names are different network name

I dont want it show `ZeroTier` instead of `31` in my case. So title may shown as `31 - ZeroTier`

**Reply by majkinetor** [reply]
I am sure we had poup before, for ad hoc stuff. 

What happened to it ?

The guy was asking for generated servers with random names.

**Reply by VShawn** [reply]
uhh.. I don't remember the pop up。

I not sure if you mean this one showing by press <kbd>tab</kbd> key on the launcher?

![image](https://github.com/1Remote/1Remote/assets/10143738/625fa072-fc20-47fe-9191-60ecb1d92e91)

**Reply by majkinetor** [reply]
Yeah Tab... that was it.

Why not providing here other connection details like with alt. I need to change hostname for example. It makes sence to be the same as alts.

Need to document this.

**Reply by majkinetor** [reply]
And it should show current settings so u can tweak it, rather then empty boxes.

**Reply by VShawn** [reply]
This is to create a new temporary session, which is why it is an empty frame. This input box has no connection with the current existing connection in 1RM.

The feature is inspired by:

![image](https://github.com/1Remote/1Remote/assets/10143738/50168a7a-3f0a-4a5e-a35a-748b14f1cb44)
